modules/auxiliary/gather/kerberoast.rb

```diff
@@ -0,0 +1,218 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::Kerberos::Client
+  include Msf::Exploit::Remote::LDAP
+  include Msf::Exploit::Remote::LDAP::Queries
+  include Msf::OptionalSession::LDAP
+
```

Comment:
Given that this PR removes the `modules/auxiliary/gather/get_user_spns.py` module in its entirety, this module should use `Msf::Exploit::Deprecated`:

```suggestion
  include Msf::Exploit::Deprecated

  moved_from 'auxiliary/gather/get_user_spns'
```

... although I'm not sure we've ever tested this code pattern with a Python module.

Reply:
Added, and seems to work as expected. Thanks!